src/components/SettlementButton/AnimatedSettlementButton.tsx

```diff
@@ -48,26 +50,28 @@ function AnimatedSettlementButton({isPaidAnimationRunning, onAnimationFinish, is
         paymentCompleteTextScale.value = 0;
         paymentCompleteTextOpacity.value = 1;
         height.value = variables.componentSizeNormal;
-    }, [buttonScale, buttonOpacity, paymentCompleteTextScale, paymentCompleteTextOpacity, height]);
+        buttonMarginTop.value = styles.expenseAndReportPreviewTextButtonContainer.gap;
+    }, [buttonScale, buttonOpacity, paymentCompleteTextScale, paymentCompleteTextOpacity, height, buttonMarginTop, styles.expenseAndReportPreviewTextButtonContainer.gap]);
 
     useEffect(() => {
         if (!isPaidAnimationRunning) {
             resetAnimation();
             return;
         }
         // eslint-disable-next-line react-compiler/react-compiler
-        buttonScale.value = withTiming(0, {duration: CONST.ANIMATION_PAY_BUTTON_DURATION});
-        buttonOpacity.value = withTiming(0, {duration: CONST.ANIMATION_PAY_BUTTON_DURATION});
-        paymentCompleteTextScale.value = withTiming(1, {duration: CONST.ANIMATION_PAY_BUTTON_DURATION});
+        buttonScale.value = withTiming(0, {duration: CONST.ANIMATION_PAID_DURATION});
+        buttonOpacity.value = withTiming(0, {duration: CONST.ANIMATION_PAID_DURATION});
+        paymentCompleteTextScale.value = withTiming(1, {duration: CONST.ANIMATION_PAID_DURATION});
 
         // Wait for the above animation + 1s delay before hiding the component
-        const totalDelay = CONST.ANIMATION_PAY_BUTTON_DURATION + CONST.ANIMATION_PAY_BUTTON_HIDE_DELAY;
+        const totalDelay = CONST.ANIMATION_PAID_DURATION + CONST.ANIMATION_PAID_BUTTON_HIDE_DELAY;
         height.value = withDelay(
             totalDelay,
-            withTiming(0, {duration: CONST.ANIMATION_PAY_BUTTON_DURATION}, () => runOnJS(onAnimationFinish)()),
+            withTiming(0, {duration: CONST.ANIMATION_PAID_DURATION}, () => runOnJS(onAnimationFinish)()),
```

Comment:
Small thing I'm just noticing is that we're just using `withTiming`, but we could use `withSpring` and/or `withBounce` to give these animations a more natural feel.

Reply:
we talked in slack about using bounce for showing the checkmark